File: app/src/main/java/com/example/myapplication/kt/presenter/TianQiContract.java
package com.example.myapplication.kt.presenter;

import com.example.myapplication.base.BaseView;
import com.example.myapplication.bean.WeatherBean;

public class TianQiContract {

     public  interface  TianQiView extends BaseView{
           void showTianQiList(WeatherBean weatherBean);
     }

    interface TianQiPresenter {
        void getWeatherList(String key,String city);
    }
}

File: app/src/main/java/com/example/myapplication/kt/presenter/TianQiContract.kt
package com.example.myapplication.kt.presenter

import com.example.myapplication.base.BaseView
import com.example.myapplication.bean.WeatherBean

class TianQiContract {
    interface TianQiView : BaseView {
        fun showTianQiList(weatherBean: WeatherBean?)
    }

    internal interface TianQiPresenter {
        fun getWeatherList(key: String?, city: String?)
    }
}